SparkPost/2web2ui
# AC-933: Updated billing endpoint

### What Changed
 - Billing info retrieves at `/billing` instead of `/account/billing`

### How To Test
 - Go to `/account/billing` or `/account/billing/plan` and check that account billing info is retrieved properly. 
 - Check that plan upgrades still work

### To Do
